src/Agent.Worker/Build/BuildJobExtension.cs

```diff
@@ -143,17 +142,17 @@ public override void InitializeJobExtension(IExecutionContext executionContext,
                 throw new Exception(StringUtil.Loc("SupportedRepositoryEndpointNotFound"));
             }
 
-            executionContext.Debug($"Primary repository: {repoInfo.Repository.Properties.Get<string>(Pipelines.RepositoryPropertyNames.Name)}. repository type: {repoInfo.Repository.Type}");
+            executionContext.Debug($"Primary repository: {repoInfo.TriggeringRepository.Properties.Get<string>(Pipelines.RepositoryPropertyNames.Name)}. repository type: {repoInfo.TriggeringRepository.Type}");
```

Comment:
Do we want to change this debug text from "Primary repository" given that it now means something different?

Reply:
absolutely, I will fix it